include/openspace/rendering/renderable.h

```diff
@@ -79,6 +79,8 @@ class Renderable : public properties::PropertyOwner {
     double boundingSphere() const;
     double interactionSphere() const;
 
+    std::string typeAsString() const;
```

Comment:
This can probably return `std::string_view` as the source will live for a long time and that way we don't need to make a copy every time we ask for it